examples/main/data-samples.js

```diff
@@ -16,6 +16,15 @@ export const meterPoints = pointGrid(1e3, [-5000, -5000, 5000, 5000]);
 
 export const worldGrid = pointsToWorldGrid(points, 0.5);
 
+export const zigzag = [
+  {path: (new Array(12)).fill(0).map(
```

Comment:
Yes you are right, this one keeps getting me. Even though the length is reported as 4, there are no values and the iteration doesn't happen. 

I do know that we don't need the parenthesis though, i.e `new Array(12).fill(0).map` works fine and looks cleaner to me. It is also possible to drop the `new` for extra terseness but I personally prefer to have it.